backend/api/v1/endpoints/auth.py

```
@@ -61,15 +53,25 @@ async def login(form_data: OAuth2PasswordRequestForm = Depends(), db: Session =
 
 @router.get("/users/me/")
 
-async def get_current_user(token: str = Depends(oauth2_scheme)):
+async def get_current_user(token: str = Depends(oauth2_scheme), db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_current_user`

(ANN201)